app/assets/stylesheets/landing_page.scss

```diff
@@ -0,0 +1,86 @@
+.overlay-container {
+  position: relative;
```

Comment:
Properties should be ordered background, height, overflow, padding-left, position